Source/LibationWinForms/grid/ImageDisplay.cs

```diff
@@ -0,0 +1,118 @@
+﻿using FileLiberator;
+using LibationFileManager;
+using System;
+using System.Drawing;
+using System.IO;
+using System.Windows.Forms;
+
+namespace LibationWinForms
+{
+	public partial class ImageDisplay : Form
+	{
```

Comment:
All the weird overrides in here are to make the form's aspect ratio always match the picture's aspect ratio.

Reply:
Ah. Can you add that comment in-code somewhere here? Otherwise I'm bound to forget